src/helpers/slack.ts

```diff
@@ -29,45 +29,29 @@ export function getEmojisToReactWith(text: string): Array<string> {
 }
 
 function validateIsEmojisDict(dict: any): boolean {
-    if (typeof dict !== "object" || dict === null) return false;
+    if (typeof dict !== "object" || dict === null) {
+        return false;
+    }
 
     for (const key in dict) {
-        if (typeof key !== "string") return false;
-        if (!Array.isArray(dict[key])) return false;
-
-        for (const elem of dict[key]) {
-            if (typeof elem !== "string") return false;
+        if(Array.isArray(dict[key])) {
```

Comment:
Is this missing a check - `if key is not a string return False` ?

Reply:
I think that will be covered by line 43.